lldb/tools/lldb-dap/README.md

```diff
@@ -1,71 +1,30 @@
-file that defines how your program will be run. The JSON configuration file can contain the following `lldb-dap` specific launch key/value pairs:
-
-|parameter          |type|req |         |
-|-------------------|----|:--:|---------|
-|**name**           |string|Y| A configuration name that will be displayed in the IDE.
-|**type**           |string|Y| Must be "lldb-dap".
-|**request**        |string|Y| Must be "launch".
-|**program**        |string|Y| Path to the executable to launch.
-|**args**           |[string]|| An array of command line argument strings to be passed to the program being launched.
-|**cwd**            |string| | The program working directory.
-|**env**            |dictionary| | Environment variables to set when launching the program. The format of each environment variable string is "VAR=VALUE" for environment variables with values or just "VAR" for environment variables with no values.
-|**stopOnEntry**    |boolean| | Whether to stop program immediately after launching.
-|**initCommands**   |[string]| | LLDB commands executed upon debugger startup prior to creating the LLDB target. Commands and command output will be sent to the debugger console when they are executed.
-|**preRunCommands** |[string]| | LLDB commands executed just before launching after the LLDB target has been created. Commands and command output will be sent to the debugger console when they are executed.
-|**stopCommands**   |[string]| | LLDB commands executed just after each stop. Commands and command output will be sent to the debugger console when they are executed.
-|**launchCommands** |[string]| | LLDB commands executed to launch the program. Commands and command output will be sent to the debugger console when they are executed.
-|**exitCommands**   |[string]| | LLDB commands executed when the program exits. Commands and command output will be sent to the debugger console when they are executed.
-|**terminateCommands** |[string]| | LLDB commands executed when the debugging session ends. Commands and command output will be sent to the debugger console when they are executed.
-|**sourceMap**      |[string[2]]| | Specify an array of path re-mappings. Each element in the array must be a two element array containing a source and destination pathname.
-|**debuggerRoot**   | string| |Specify a working directory to use when launching lldb-dap. If the debug information in your executable contains relative paths, this option can be used so that `lldb-dap` can find source files and object files that have relative paths.
-
-### Attaching Settings
-
-When attaching to a process using LLDB you can attach in a few ways
+By default, the VS-Code extension will expect to find `lldb-dap` in your `PATH`.
+Alternatively, you can explictly specify the location of the `lldb-dap` binary using the `lldb-dap.executable-path` setting.
```

Comment:
Is `lldb-dap.executable-path` a setting in VSCode?

Reply:
yes, this is one of the VS-Code settings added by this extension